src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -193,11 +193,7 @@ m_cursorGrabbed   (m_fullscreen)
 
     // Choose the window style according to the Style parameter
     DWORD win32Style = WS_VISIBLE;
-    if (style == Style::None)
-    {
-        win32Style |= WS_POPUP;
-    }
-    else
+    if (style != Style::None)
```

Comment:
No need to test explicitly against Style::None, in this case none of the tests below will succeed and the `win32Style` variable will correctly stay at its initial value.